examples/Supervised/VersatileClassificationTutorial-LDA.tpp

```diff
@@ -1,54 +0,0 @@
-
```

Comment:
Why is this removed? LDA is a very useful method. It is particularly nice for ML beginners, because it does not have any parameters.

Reply:
LDA is covered in the getting started tutorial.